src/dstack/_internal/core/backends/aws/compute.py

```diff
@@ -94,6 +97,64 @@ def terminate_instance(
             else:
                 raise e
 
+    def create_instance(
```

Comment:
Logic is duplicated in `create_instance()` and `run_job()`. Should be refactored to avoid duplication.

Reply:
I've removed logic duplicates